### ISSUE TITLE
Bluetooth doesn't start without rebooting the device

### ISSUE THREAD

**Issue by stanwood77** [ubports/ubuntu-touch#1705]
**Bluetooth doesn't start without rebooting the device**

- Device: Lavender
- Channel: OTA-16 stable

### Steps to reproduce

Start Bluetooth via the top control panel, or via settings

### Expected behavior

- Bluetooth is activated 
- Bluetooth icon lights up
- Device is discoverable

### Actual behavior

- Bluetooth bottom is on
- Bluetooth top icon remains unlit
- Device is not discoverable
- Device is not able to pair, send nor receive files
- Restart the device gets the Bluetooth on and makes it work normally
 
Note: In car phone calls however do not work as calls are not redirected to the car Bluetooth device

### Logfiles and additional information

rfkill list returns:

`0: bt_power: Bluetooth

            Soft blocked: yes
            Hard blocked: no`

**Comment by Flohack74** [comment]
So did you turn off Bluetooth and it wont turn back on?

**Comment by stanwood77** [comment]
If I turn Bluetooth off, no change
If I turn Bluetooth on again, no change
When I restart the device, the Bluetooth will be automatically well activated. Then I can turn it off / on again, everything works.
So basically, if I need Bluetooth, I turn it on, restart the device, and it works.

**Comment by Flohack74** [comment]
Yeah probably handling of rfkill is missing, thats a matter of the device porter. I dont know from the top of my head who would that be though.

**Comment by stanwood77** [comment]
Indeed. Thank you. 
I think the porter of Xiaomi devices is Danct12

**Comment by Danct12** [comment]
> Note: In car phone calls however do not work as calls are not redirected to the car Bluetooth device

This might be a upstream issue.. will look into it.

**Comment by Flohack74** [comment]
Yeah carkits ar hit or miss basically. Not much we can do, other than trying to upgrade Bluez stack. Will come with 20.04 latest.
